src/landing/model_list.rs

```diff
@@ -83,6 +83,7 @@ impl Widget for ModelList {
         if self.loading_delay.is_event(event).is_some() {
             self.update_loading_and_error_message(cx, scope);
         }
+        self.redraw(cx);
```

Comment:
Asking the framework to redraw the widget when *any* event happens could lead poor performance, just because ir could be redrawing most of the time (for instance, any mouse move event will trigger a redraw here). Ideally, we should invoke `self.redraw(cx)` when certain condition is met.

Reply:
@jmbejar Thanks for your reviewing. Also, I have a question about issue 132, it has already been merged. I think the arrow icon shouldn't appear when there are no available models, like this.
![image](https://github.com/user-attachments/assets/c68c81f5-93a9-4ebe-9e46-e64a9ee18995)
I don't know if this situation is the minor issue you mentioned.

Reply:
> @jmbejar Thanks for your reviewing. Also, I have a question about issue 132, it has already been merged. I think the arrow icon shouldn't appear when there are no available models

This issue is something I was not aware... so feel free to drop a PR to fix this problem, if you want 👍

Reply:
I would be more than happy to fix this problem. I will try to solve it.